generators/Exercise.fs

```diff
@@ -57,6 +61,19 @@ type GeneratorExercise() =
         |> this.MapCanonicalData
         |> this.Render  
         |> this.WriteToFile
+        
+    member this.ReadVersion () =
+        (*
+            Read from the top of the file e.g.
+            "// This file was auto-generated based on version 1.2.0 of the canonical data."
+        *)
+        let isNum c = c >= '0' && c <= '9'
```

Comment:
I think you could replace this with `System.Char.IsDigit`

Reply:
Ah! I knew there had to be a better way.